server/chat-plugins/scavengers.js

```diff
@@ -1543,7 +1556,7 @@ exports.commands = {
 			"- /scav resetladder - resets the monthly scavenger leaderboard. (Requires: # & ~)",
 			"- /scav setpoints [1st place], [2nd place], [3rd place], [4th place], [5th place], ... - sets the point values for the wins. Use `/scav setpoints` to view what the current point values are. (Requires: # & ~)",
 			"- /scav setblitz [value] ... - sets the blitz award to `value`. Use `/scav setblitz` to view what the current blitz value is. (Requires: # & ~)",
-			"- /scav queue <em>[host] | [hint] | [answer] | [hint] | [answer] | [hint] | [answer] | ...</em> - queues a scavenger hunt to be started after the current hunt is finished. (Requires: % @ * # & ~)",
+			"- /scav queue(unrated) <em>[host] | [hint] | [answer] | [hint] | [answer] | [hint] | [answer] | ...</em> - queues a scavenger hunt to be started after the current hunt is finished. (Requires: % @ * # & ~)",
```

Comment:
I don't think we use the `(unrated)` syntax, it's pretty confusing here. I'd make them different commands in the list.

Reply:
What would you prefer instead? I'm just using the same syntax as the /start(mini/official/unratrd/practice)hunt help entry above)

Reply:
> What would you prefer instead?

> I'd make them different commands in the list.

Reply:
Oh, I didn't know they did that for the other ones.

Reply:
I'd at least suggest `queue(rated/unrated)` to make it clearer what's going on there.